src/azure-cli-core/azure/cli/core/tests/test_util.py

```diff
@@ -198,6 +198,140 @@ def test_can_launch_browser(self, webbrowser_get_mock, get_platform_mock):
             result = can_launch_browser()
             self.assertFalse(result)
 
+    def test_configured_default_setter(self):
+        config = mock.MagicMock()
+        config.use_local_config = None
+        with ConfiguredDefaultSetter(config, True):
+            self.assertEqual(config.use_local_config, True)
+        self.assertIsNone(config.use_local_config)
+
+        config.use_local_config = True
+        with ConfiguredDefaultSetter(config, False):
+            self.assertEqual(config.use_local_config, False)
+        self.assertTrue(config.use_local_config)
+
+    @mock.patch('azure.cli.core.__version__', '7.8.9')
+    def test_get_az_user_agent(self):
+        actual = get_az_user_agent()
+        self.assertEqual(actual, 'AZURECLI/7.8.9')
+
+    @mock.patch('azure.cli.core._profile.Profile.get_raw_token', autospec=True)
+    @mock.patch('requests.Session.send', autospec=True)
+    def test_send_raw_requests(self, send_mock, get_raw_token_mock):
```

Comment:
These test methods are moved from the wrong class `TestHandleException` to the right class `TestUtils`.